src/submission/models.py

```diff
@@ -2063,6 +2063,19 @@ def __str__(self):
             return self.name
         return f"Unnamed Section {self.pk}"
 
+    def editor_display_name(self):
```

Comment:
Why is this function called `editor_display_name` if it doesn't do anything regarding an editor role. How about `display_name_public_submission`